src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageActions.styles.ts

```diff
@@ -25,20 +25,20 @@ export const messageActionsGroup: CSSObject = {
   padding: '2px',
 };
 
-export const messageBodyActions: CSSObject = {
+export const messageBodyActions = (rightMarginWidth: number): CSSObject => ({
   display: 'inline-flex',
   alignItems: 'center',
   minHeight: '32px',
   minWidth: '40px',
   position: 'absolute',
-  right: '16px',
+  right: `${16 - rightMarginWidth}px`,
```

Comment:
I need to position reactions menu to look in the old way - 16px from the right of the screen, not of the message container. I can't use `position: fixed` as I need it only horizontally, and vertically it should still be relative to the parent, to be scrolled. I also tried to play with `left: (100vw - 100% - 16px)`, but `100%` is a container, resized to the right margin of the `MessageList` container.